learn/gettingStarted.html.haml

```diff
@@ -0,0 +1,185 @@
+      include::learn/explore.adoc[]
+
+      == Technical details
+
+      By default everything runs using H2 database with file storage but this image also enables you to switch to other databases like MySQL or PostgreSQL.
+      All dependencies and data source configurations are done automatically, you just need to specify a set of environment variables.
+      Alternatively, you can use the provided https://docs.docker.com/compose/install/[Docker Compose] examples to get started using an alternative database.
+      When using the provided Docker Compose examples, all dependencies to get a running container and configuration for the target database is already set in place.
+
+      The Docker Compose examples are located in the following repository: https://github.com/jboss-dockerfiles/jbpm/tree/master/docker-compose-examples
+
+      === MySQL Example using Docker Compose
+
+        docker-compose -f docker-compose-examples/jbpm-full-mysql.yml up
+
+      === MySQL Example using remote connection
+
+        docker run -p 8080:8080 -p 8001:8001 -d --name jbpm-server-full -e JBPM_DB_DRIVER=mysql -e JBPM_DB_HOST=172.17.0.1 jboss/jbpm-server-full:latest
+
+      === PostgreSQL Example using Docker Compose
+
+        docker-compose -f docker-compose-examples/jbpm-full-postgres.yml up
+
+      === PostgreSQL Example using remote connection
+
+        docker run -p 8080:8080 -p 8001:8001 -d --name jbpm-server-full -e JBPM_DB_DRIVER=postgres -e JBPM_DB_HOST=172.17.0.1 jboss/jbpm-server-full:latest
+
+      === Environment variables
+
+      * `JBPM_DB_DRIVER` = Specify which database driver to use. Allows either: 'h2', 'mysql' or 'postgres'. Default: 'h2'.
+      * `JBPM_DB_HOST` = Specify hostname of the database. Default: 'localhost'
+      * `JBPM_DB_PORT` = Specify port of the database. Default: '3306' if using 'mysql' driver or '5432' in case of 'postgres'.
+      * `JBPM_DB_NAME` = Specify name of the database to use. Default: 'jbpm'
+      * `JBPM_DB_USER` = Specify user to use to authenticate to the database. Default: 'jbpm'
+      * `JBPM_DB_PASSWORD` = Specify user's password to use to authenticate to the database. Default: 'jbpm'
+
+      === Useful Docker commands
+
+      === View container logs
+        $ docker logs -f jbpm-server-full
+
+      ==== List running containers
+        $ docker ps
+        CONTAINER ID        IMAGE                           COMMAND                CREATED             STATUS              PORTS                                            NAMES
+        88f2e110aff5        jboss/jbpm-server-full:latest   "./start_jbpm-wb.sh"   7 minutes ago       Up 7 minutes        0.0.0.0:8001->8001/tcp, 0.0.0.0:8080->8080/tcp   jbpm-server-full
+
+      or
+
+        $ docker ps -a
+
+      In case you want to also list stopped containers.
+
+      ==== Stopping the jBPM Server Full Distribution container
+        $ docker stop jbpm-server-full
+
+      ==== Delete jBPM Server Full Distribution container and all itś data
```

Comment:
I see you started to use polish character :)

> itś

Reply:
trying to get some points for my PR :P